hpx/components/containers/partitioned_vector/detail/make_index_sequence.hpp

```diff
@@ -0,0 +1,113 @@
+//  Copyright (c) 2017 Antoine Tran Tan
```

Comment:
This is a duplication of code that can be found here:
https://github.com/STEllAR-GROUP/hpx/blob/master/hpx/util/detail/pack.hpp#L44